core/domain/stats_jobs_one_off.py

```diff
@@ -370,26 +390,17 @@ def reduce(exp_id, stringified_values):
                 # Log exp_id, exp_version, state_names, state_stats_mapping.
                 for answer_group in init_state.interaction.answer_groups:
                     dest_state = answer_group.outcome.dest
+                    # Explicit handling of the pseudo-END state, where we
```

Comment:
Some older explorations had the pseudo-END state as a potential destination from the initial state. We skip the pseudo-END state for the purposes of this calculation.

Reply:
Done.